<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>depositcalc</class>
 <widget class="QWidget" name="depositcalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>251</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QCustomPlot" name="graph" native="true">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>60</y>
     <width>451</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>151</x>
     <y>180</y>
     <width>114</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>дата в</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="convert_to_comboBox_2">
      <item>
       <property name="text">
        <string>днях</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>месяцах</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>годах</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="input_show">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Срок</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="res_monthly_show">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>250</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>переплата </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>30</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Помесячная выплата</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ставка</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Сумма</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="data_show">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>180</y>
     <width>133</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Период</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="type_comboBox_2">
      <item>
       <property name="text">
        <string>в конце срока</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>каждый день</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>каждую неделю</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>раз месяц</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>раз в квартал</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>раз в полгода</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>раз в год</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="overpayment_show">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>290</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>260</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ежемесячный</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="total_payout_show">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>330</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="percent_show">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>общ. выплата</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="percent_show_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Налог%</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
